### PR TITLE
Fix miner tab thread slider

### PR DESCRIPTION
**Pull request by unclejam**
Fix miner tab thread slider

### CODE REVIEW

File: src/qt/miningpage.cpp [hunk 1]
```diff
@@ -11,18 +11,32 @@
 #include <boost/thread.hpp>
 #include <stdio.h>
 
+
+static int MaxTreads() {
```
Comment (Duality-CDOO) [inline src/qt/miningpage.cpp]: typo? of MaxThreads as MaxTreads.

**Comment by unclejam** [comment]
Yes, it was a typo and I re-based the commit with the correction.

**Comment by Duality-CDOO** [comment]
Probably good to have a check for dynodeSync.IsBlockchainSynced() and if it is not synced, do not allow mining and show a message to the user saying "Blockchain/Sporks not Synced, Mining Disabled". Or similar.

**Comment by Duality-CDOO** [comment]
ACK on the PR as it stands currently.

**Comment by Duality-CDOO** [comment]
I'll merge this work as it fixes the known issues. The message to users is added functionality and can be a seperate PR.

**Pull request merged**